src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/Animation/SequentialAnimationsPage.xaml.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace UITests.Shared.Windows_UI_Xaml_Media.Animation
+{
+	[SampleControlInfo("Animations", "Sequential Animations")]
+	public sealed partial class SequentialAnimationsPage : Page
+	{
+		private readonly TimeSpan DURATION = TimeSpan.FromMilliseconds(1000);
```

Comment:
Shorter is better since it slows down the CI build... 100 ms should be enough to validate it doesn't crash

Reply:
But this page is also shown in the samples application

Reply:
I'll shorten it to a half 2nd at least, you're right 1000 is too long